go.mod

```diff
@@ -1,35 +1,104 @@
 module github.com/aspenmesh/istio-vet
 
-go 1.13
+go 1.17
+
+replace (
+	// pseudo version for tag 1.11.4-am1-rc0
+	istio.io/istio => github.com/aspenmesh/istio-private v0.0.0-20211110174632-2408efb7dad5
```

Comment:
I'm not thrilled with having our closed source fork of Istio as a dependency in an open source project but, if that's what we have to do to make it work, then that's what we gotta do.